package Day10;

import java.util.Scanner;

public class MultipleExceptionHandling {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		System.out.println("Program is started");
		System.out.println("Program is running");
		
		Scanner input = new Scanner(System.in);
		
		int a = 100;
		try {
			System.out.println(a/0);
		}
		catch(ArithmeticException e) {

			System.out.println("exception occured");
		}		catch(NullPointerException e) {

			System.out.println("exception occured");
		}		catch(NumberFormatException e) {

			System.out.println("exception occured");
		}
	}System.out.println("Program is finished");


}
